docker-compose.infra.yml

```
@@ -0,0 +1,19 @@
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=userdb
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    deploy:
+      resources:
+        limits:
+          memory: 500m
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)